roles/wazuh/ansible-wazuh-manager/defaults/main.yml

```diff
@@ -16,7 +16,6 @@ wazuh_manager_sources_installation:
   user_language: "en"
   user_no_stop: "y"
   user_install_type: "server"
-  user_dir: "/var/ossec"
```

Comment:
Hi @sergiogp98,

Same as in the Wazuh agents default variables case, this variable is related to sources installation, I think it would a good idea to keep it despite it is currently unused.

Reply:
Fixed 0d778f78850592eb178795aa18a5aef2dfb11ea9 @Zenidd